docs/nodejs/nodejs-tutorial.md

```diff
@@ -166,7 +166,7 @@ You can also write code that references modules in other files. For example, in
 
 ## Debugging your Express Application
 
-You will need to create a debugger configuration file `launch.json` for your Express application. Click on the Debug icon in the **Activity Bar** and then the Configure gear icon at the top of the Debug view to create a default `launch.json` file.  Select the **Node.js** environment.  When the file is first created, VS Code will look in `package.json` for a `start` script and will use that value as the `program` (which in this case is `"${workspaceFolder}\\bin\\www`) for the **Launch Program** configuration.
+You will need to create a debugger configuration file `launch.json` for your Express application. Click on the Debug icon in the **Activity Bar** and then the Configure gear icon at the top of the Debug view to create a default `launch.json` file.  Select the **Node.js** environment by ensuring that the `type` property in `configurations` is set to `"node"`.  When the file is first created, VS Code will look in `package.json` for a `start` script and will use that value as the `program` (which in this case is `"${workspaceFolder}\\bin\\www`) for the **Launch Program** configuration.

```

Comment:
"Select" by itself implies there is a list to choose from. Added wording how to make the selection.

Reply:
There used to be a dropdown where the user would select **Node.js** from a list. I don't see it in the latest build. Thanks for noticing.